app/views/embed/template/_file.html.erb

```diff
@@ -4,4 +4,16 @@
   <%= render 'metadata_panel', viewer: viewer %>
   <%= render 'embed_this/file', viewer: viewer %>
   <%= render 'footer/generic', viewer: viewer %>
- </div>
+  <script src="<%= asset_url('tree.js') %>"></script>
+  <script>
+    document.onreadystatechange = () => {
+      if (document.readyState === "complete") {
```

Comment:
was it determined that `addEventListener('load', () => ...` would definitely not work here?

Reply:
Can this go back in the tree.js now that this is not being loaded via jQuery.ajax?